docs/tutorial/development-environment.md

```diff
@@ -0,0 +1,114 @@
+# Developer Environment
+
+Electron development is essentially Node.js development. To turn your operating
+system into an environment capable of building desktop apps with Electron,
+you will merely need Node.js, npm, a code editor of your choice, and a
+rudimentary understanding of your operating system's command line client.
+
+## Setting up macOS
```

Comment:
shouldn't there be `on` here? Setting up **on** macOS, Setting up **on** Windows, etc.

Reply:
The point is: "Setting up Windows _for Electron development_" or "Setting up Windows _as a developer environment for Electron_.